internal/bundle/bundle.go

```diff
@@ -44,7 +44,11 @@ func Validate(ctx context.Context, imagePath string) (*Report, error) {
 	}
 	validators := validation.DefaultBundleValidators.WithValidators(
 		validation.AlphaDeprecatedAPIsValidator,
-		validation.OperatorHubValidator,
+		// OperatorHubV2Validator, StandardCapabilitiesValidator and StandardCategoriesValidator provide the same functionality
+		// as the deprecated OperatorHubValidator validator
```

Comment:
Nit: We probably don't need these comments here. It was useful for my review, but I don't think it needs to live in the source.

Reply:
Agreed. Should just live in the commit message.